.github/workflows/ci.yml

```diff
@@ -41,8 +41,6 @@ jobs:
         run: |
           cd build
           cpanm --installdeps --local-lib ${GITHUB_WORKSPACE}/local --with-all-features --with-develop --with-suggests .
-      - name: Install
-        run: cpanm Devel::Cover::Report::Coveralls
```

Comment:
Any particular reason why this change is a part of this PR?

Reply:
A few lines up, `Devel::Cover::Report::Coveralls` is already installed. It looked like a merge conflict, so I corrected it. Apparently, I was mistaken, resulting in this superfluous change. Do you want me to revert?

Reply:
Then it is good. I just thought the change stood out